src/charmed_analytics_ci/templates/pr_body.md.j2

```diff
@@ -0,0 +1,42 @@
+This PR was opened automatically by the `charmed-analytics-ci` library as part of the Rock CI system after the rock image was built and published.
```

Comment:
suggestion(non-blocking): link the PR that did the rock update in the PR description. I'm usually doing that in integrate PRs for easier traceability. It's non-blocking because we can trace it with the commit hash so I'm ok going without it.

Reply:
I'm +1 to point to the commit, @kimwnasptd by the value of the image do you mean the registry/name:tag ? that's already in the PR title tho

Reply:
aah you are right! Then, the rock in the title SGTM!

Reply:
ok added the pr info to the body. :)

Reply:
@misohu small question: I see that now the `triggering_pr` is an arg. How will this arg be filled?

I understand that the workflow files in the rock repos will be filling this value. But, since the workflow will be getting triggered by the merged commit, will the plan be to be extracting the PR url in those actions?

Reply:
sample pr https://github.com/canonical/test-kubeflow-automation/pull/138

Reply:
worst case i will just send commit link only

Reply:
OK sounds good! Mostly wanted to ensure we had a plan A at least in mind for it, or whether we'd need to rename this from `triggering_pr` to `triggering_commit`. 

Since we do, LGTM